### PR TITLE
Summary 2 of Mish

### PR DESCRIPTION
**Pull request by biofizzatreya**
Submission for summary 2 of mish paper

### CODE REVIEW

File: _summaries/misra2019mish_2.md [hunk 1]
```diff
@@ -1,19 +1,34 @@
...
+author: biofizzatreya
+score: 7
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+This paper attempts to fix the vanishing gradient problem of Relu activation functions by using, what they call Mish. Mish is defined as $$ x\cdot tanh\left(\text{ln}(1+e^x)\right) $$. The authors compare the effect of using different activation functions such as Relu, Swish and Mish.
```
Comment (dalmeraz) [inline _summaries/misra2019mish_2.md:11]: I think it would be good to expand some of the motivations of Mish and why they're important here:
* non-monotonicity
* ability to preserve small negative weights: "fix the vanishing gradient problem"
* smooth profile: related to differentiability. Allows for smoother gradients.

File: _summaries/misra2019mish_2.md [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+This paper attempts to fix the vanishing gradient problem of Relu activation functions by using, what they call Mish. Mish is defined as $$ x\cdot tanh\left(\text{ln}(1+e^x)\right) $$. The authors compare the effect of using different activation functions such as Relu, Swish and Mish. 
+
+![Picture1](https://user-images.githubusercontent.com/13065170/132050780-f3438c1c-a2b6-433d-a3de-2d9279d81cce.png)
```
Comment (dalmeraz) [inline _summaries/misra2019mish_2.md:13]: Unsure comment: Not sure if it's okay to include the explanation sections for the tables since in the summary guide it says "use your own words (do not copy from the paper)". Maybe it would be better to explain those sections on your own, its your call.

File: _summaries/misra2019mish_2.md [hunk 3]
```diff
@@ -1,19 +1,34 @@
...
+![Picture4](https://user-images.githubusercontent.com/13065170/132052717-1e3aa06d-364c-4d74-b705-a0e057a1e30c.png)
+
+
+
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* A differentiable activation function Mish
```
Comment (dalmeraz) [inline _summaries/misra2019mish_2.md:32]: Maybe "smooth" is a better key word as opposed to "differentiable" since it seems to be a more direct motivation.

File: _summaries/misra2019mish_2.md [hunk 4]
```diff
@@ -1,19 +1,34 @@
...
+
+Understandably Mish is differentiable and not as rough compared to Relu
+
+![Picture2](https://user-images.githubusercontent.com/13065170/132051192-10e6a969-44fb-42b1-b1b2-ff45573533d1.png)
+
+Mish compares well with other activation functions.
+
+![Picture3](https://user-images.githubusercontent.com/13065170/132051510-42e39ced-5bb2-4589-a94c-fcef71cd8077.png)
+
+Mish seems to work quite well when it comes to overfitting. Compared to other activation functions, Mish does not increase its loss as much as number of layers are increased. However it also loses accuracy when it comes to corrupted training data. They have performed other comparisons as well. To distinguish itself from Swish, they show that Swish
```
Comment (dalmeraz) [inline _summaries/misra2019mish_2.md:23]: You could explain what corrupted data is.

File: _summaries/misra2019mish_2.md (hunk 1 above)
Comment (dalmeraz) [reply _summaries/misra2019mish_2.md:11]: I think it would be useful to explain the "vanishing gradient" problem of ReLU by paraphrasing  "Dying ReLU ... is experienced through a gradient information loss caused by collapsing the negative inputs to zero"  which will further motivating Mish's "ability to preserve small negative weights".

Comment (dalmeraz) [reply _summaries/misra2019mish_2.md:11]: Since Mish is so closely related to Swish, it might be good to mention this relation and elaborate further when presenting the activation graphs.

File: _summaries/misra2019mish_2.md [hunk 5]
```diff
@@ -1,19 +1,34 @@
...
+![Picture2](https://user-images.githubusercontent.com/13065170/132051192-10e6a969-44fb-42b1-b1b2-ff45573533d1.png)
+
+Mish compares well with other activation functions.
+
+![Picture3](https://user-images.githubusercontent.com/13065170/132051510-42e39ced-5bb2-4589-a94c-fcef71cd8077.png)
+
+Mish seems to work quite well when it comes to overfitting. Compared to other activation functions, Mish does not increase its loss as much as number of layers are increased. However it also loses accuracy when it comes to corrupted training data. They have performed other comparisons as well. To distinguish itself from Swish, they show that Swish 
+decreases Top-1 accuracy in larger models, unlike Mish. Mish was also able to seriously reduce the runtime for object detection training, possibly due to its differentiability and lack of conditional statements.
+
+![Picture4](https://user-images.githubusercontent.com/13065170/132052717-1e3aa06d-364c-4d74-b705-a0e057a1e30c.png)
```
Comment (dalmeraz) [inline _summaries/misra2019mish_2.md:26]: Since softplus is part of the Mish formula it might be good to define it at least mathematically.

**Review by dalmeraz — COMMENTED** [review]
Most comments are in general stuff to add. Feel free to add what you feel might be useful from the comments, only length constraint we have is for it to be "<5min to read".
